### DISCUSSION TITLE
RES8 crash before loading upscaler plugin

### DISCUSSION THREAD

**Discussion by kazmi789** [praydog/REFramework#1046] [General Questions]
can someone guide me how to use upscaler in RES8 ... i tried files from pd scaler which around 35mb and put the file inside with base plugin but game crash when loading loading plugins.

**Comment by GennadijK1987** [comment]
intall drivers 531.41 if you have nvidia video card

**Reply by kazmi789** [reply]
i have AMD rx 6600xt with new updated drivers
